packages/graphiql-examples/cdn/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "graphiql-example-cdn",
+  "private": true,
```

Comment:
We either need to not publish this (`"private": true`), or we need to give it a version number. Is this intended to be published to NPM?

Reply:
It should be private: true